pkg/kubelet/managed/managed_test.go

```diff
@@ -0,0 +1,275 @@
+package managed
+
+import (
+	"testing"
+
+	v1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/resource"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+func TestStaticPodManaged(t *testing.T) {
+	testCases := []struct {
+		pod                 *v1.Pod
+		expectedAnnotations map[string]string
+	}{
+		{
+			pod: &v1.Pod{
+				TypeMeta: metav1.TypeMeta{
+					Kind:       "Pod",
+					APIVersion: "",
+				},
+				ObjectMeta: metav1.ObjectMeta{
+					Name:      "test",
+					UID:       "12345",
+					Namespace: "mynamespace",
+					Annotations: map[string]string{
+						"workload.openshift.io/management": "",
+					},
+				},
+				Spec: v1.PodSpec{
+					Containers: []v1.Container{
+						{
+							Name:  "image",
```

Comment:
nit: Could change the container name to be something other than `image`.

Reply:
fixed to 'nginx'